scenarios/kubernetes_e2e.py

```diff
@@ -230,6 +230,7 @@ def add_environment(self, *envs):
         Host-specific environment variables are ignored."""
         # TODO(krzyzacy) change this to a whitelist?
         docker_env_ignore = [
+            'E2E_PUBLISH_PATH',  # Use --publish flag
```

Comment:
somewhere like https://github.com/krzyzacy/test-infra/blob/76a661fc7c58b9c48ef3e526683cde9a0cd9a885/jenkins/bootstrap_test.py#L1967-L1973 might be a better place to assert the env is not there anymore